parametric/test_otel_tracer.py

```diff
@@ -0,0 +1,13 @@
+from parametric.spec.otel_trace import find_otel_span_in_traces, OtelSpan
+
+
+def test_start_otel_api_span_with_set_name(test_agent, test_library):
+    """Test span is created with OTel API of Datadog Tracing Library"""
+    with test_library:
+        with test_library.start_otel_span(name="operation", service="my-webserver", resource="/endpoint",) as parent:
+            parent.set_attributes({"key": "val"})
+            parent.set_name("new_op")
+
+    span = find_otel_span_in_traces(test_agent.wait_for_num_traces(1), OtelSpan(name="web.otel"))
```

Comment:
Would this work? 
```suggestion
    span = test_agent.wait_for_num_traces(1)[0]
    assert span.get_name() == “new_op”
```

Reply:
I have updated this to `span = get_span(test_agent)`